guides/doc-Upgrading_and_Updating/topics/migrating_a_self_registered_satellite.adoc

```diff
@@ -32,7 +32,7 @@ To remove the `katello-agent`, stop and disable the `goferd` service, then enter
 ----
 # systemctl stop goferd
 # systemctl disable goferd
-# yum remove katello-agent
+# {package-remove} katello-agent
```

Comment:
This code block definition needs to resolve attributes, e.g. using `[options="nowrap", subs="+quotes,verbatim,attributes"]`.